test-utils/throttleMock.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+  var queue = []
+  return {
+    throttle: function(fn) {
```

Comment:
Or do you mean the closure that's returned?

It is to emulate the `throttle` API. Where a throttled function can be scheduled several times, but runs only once, on `fire` (which emulates the next frame).